diff --git a/trafgen_parser.y b/trafgen_parser.y
index 71f5ddd..dd7133c 100644
--- a/trafgen_parser.y
+++ b/trafgen_parser.y
@@ -594,7 +594,7 @@ int compile_packets(char *file, int verbose, int cpu, bool invoke_cpp)
 
                slprintf(tmp_file, sizeof(tmp_file), "%s/.tmp-%u-%s", dir, rand(), base);
                slprintf(cmd, sizeof(cmd), "cpp -I" PREFIX_STRING
-                        "/etc/netsniff-ng/ %s > %s", file, tmp_file);
+                        ". %s > %s", file, tmp_file);
                system(cmd);
 
                file = tmp_file;
